openlibrary/templates/search/sort_options.html

```diff
@@ -0,0 +1,28 @@
+$def with (exclude, default)
+
+$ sort = 'sort'
+
+<span class="tools"><img src="/images/icons/icon_sort.png" alt="$_('Sorting by')" width="9" height="11" style="margin-right:10px;"/>
+    $code:
+      sort_options = [
+        { 'sort': None, 'name': _("Relevance"), 'ga_key': 'Relevance' },
```

Comment:
We'll likely want to give this a proper name here, and ...

```suggestion
        { 'sort': 'relevance', 'name': _("Relevance"), 'ga_key': 'Relevance' },
```